Controllers/window_controller.py

```diff
@@ -82,10 +83,8 @@ def __init__(self, window):
             playback_speed_combo_box.currentIndexChanged.connect(
                 self.set_playback_speed)
 
-        self.total_time_in_secs = 0
-        self.curr_time_secs = 0
-        self.curr_time_minutes = 0
-        self.curr_time_hours = 0
+        # Holds the time for the loaded video in ms.
+        self.current_time = 0
```

Comment:
Why is current time a data member of the window controller class? Could we just have that be a local variable to the get_video_time_total method?